src/utils/data-mapper.ts

```diff
@@ -8,74 +8,6 @@ import { MediaMetadataInterface } from '../models/MediaMetadata';
 import { SeasonMetadataInterface } from '../models/SeasonMetadata';
 import { SeriesMetadataInterface } from '../models/SeriesMetadata';
 
-const openSubtitlesMovieMap = {
-  'metadata.cast': [
-    {
-      key: 'actors?',
-      transform: (val): Array<string> => _.isEmpty(_.values(val)) ? null : _.values(val),
-    },
-  ],
-  'metadata.genres': [
-    {
-      key: 'genres?',
-      transform: (val): Array<string> => _.isEmpty(_.values(val)) ? null : _.values(val),
-    },
-  ],
-
-  'metadata.rating': [
-    {
-      key: 'rating',
-      transform: (val: string): number => parseFloat(val),
-    },
-  ],
-  'metadata.goofs': 'goofs',
-  'metadata.cover': 'poster',
-  'metadata.imdbid': 'imdbID',
-  'moviehash': 'osdbHash',
-  'metadata.tagline': 'tagline',
-  'metadata.title': 'title',
-  'metadata.trivia': 'trivia',
-  'metadata.duration': 'runtime',
-  'type': 'type',
-  'metadata.year': 'year',
-  'metadata.votes': 'votes',
-};
-
-const openSubtitlesEpisodeMap = {
-  'metadata.cast': [
-    {
-      key: 'actors?',
-      transform: (val): Array<string> => _.isEmpty(_.values(val)) ? null : _.values(val),
-    },
-  ],
-  'metadata.genres': [
-    {
-      key: 'genres?',
-      transform: (val): Array<string> => _.isEmpty(_.values(val)) ? null : _.values(val),
-    },
-  ],
-
-  'metadata.rating': [
-    {
-      key: 'rating',
-      transform: (val: string): number => parseFloat(val),
-    },
-  ],
-  'metadata.goofs': 'goofs',
-  'metadata.cover': 'poster',
-  'metadata.imdbid': 'imdbID',
-  'moviehash': 'osdbHash',
-  'metadata.tagline': 'tagline',
-  'metadata.episode_title': 'title',
-  'metadata.trivia': 'trivia',
-  'metadata.duration': 'runtime',
-  'type': 'type',
-  'metadata.year': 'year',
-  'metadata.votes': 'votes',
-  'metadata.season': 'season',
-  'metadata.episode': 'episode',
-};
-
 const tmdbEpisodeMap = {
```

Comment:
`tmdbEpisodeMap -> 'credits': 'credits',`
you should do the same here, for episodes :
`actors?: Array<string>;`
`directors?: Array<string>;`

`tmdbSeriesMap -> 'credits': 'credits',`
you should do the same here, for series :
`actors?: Array<string>;`
`directors?: Array<string>;`